lib/decimal.ex

```diff
@@ -289,9 +290,12 @@ defmodule Decimal do
   @doc """
   Compares two numbers numerically. If the first number is greater than the second
   `:gt` is returned, if less than `:lt` is returned, if both numbers are equal
-  `:eq` is returned. Otherwise, if any number is a `NaN`, NaN is returned.
+  `:eq` is returned.
+
+  Neither number can be a `NaN`.  If you need to handle quiet NaNs, use `compare/2`, but it returns `t` instead of
+  `:lt | :eq | :gt`.
```

Comment:
I think we should remove everything after the comma. The behavior of `compare/2` should be clear when reading the docs for it.